.buildkite/pipeline.yml

```diff
@@ -2,7 +2,7 @@
 
 env:
   SETUP_GVM_VERSION: 'v0.5.0'
-  GO_VERSION_CHOCO: "1.20.5"
+  GO_VERSION_CHOCO: "1.21.9"
```

Comment:
It looks like chocolaty is inconsistent availability for go's previouus minor releases; 1.21.9 is not available.

If we remove the go version specification for the windows tests the latest package would be used (in this case a 1.22.x release)
@cmacknz @fearful-symmetry what do you think?

Reply:
Chocolaty doesn't package every go version, so even if this works this time we could run into this again.  Can we please consider moving away from chocolaty and to something that will work for every go version?  For example https://github.com/andrewkroh/gvm has been used in the past on Windows and it works fine.

Reply:
Yeah, using `gvm` or something might be a good idea.